src/api/telemetry/TelemetryMetadataManager.js

```diff
@@ -57,13 +57,13 @@ define([
 
         if (valueMetadata.format === 'enum') {
             if (!valueMetadata.values) {
-                valueMetadata.values = _.pluck(valueMetadata.enumerations, 'value');
+                valueMetadata.values = Object.entries(valueMetadata.enumerations).reduce((a, [key, {value}]) => {a [key] = value; return a}, []);
```

Comment:
See previous comments about idiosyncratic JavaScript. Since we're doing this in several places, could we make this a util function with a descriptive name?

Reply:
Way easier to use map: `valueMetadata.values.map(x => x.name);`